src/types/gameConsole.ts

```diff
@@ -12,109 +12,583 @@ export default class GameConsole {
+    ),
+    new GameConsole(
+      /WonderSwan Color/i,
+      [".wsc"],
+      "wonderswan",
+      "WonderSwan",
+      "WS",
+      "wswanc"
+    ),
     // Bit Corporation
-    new GameConsole(/Gamate/i, [/* '.bin' */], 'gamate', 'Gamate', undefined),
+    new GameConsole(
+      /Gamate/i,
+      [
+        /* '.bin' */
+      ],
+      "gamate",
+      "Gamate",
+      undefined,
+      "gamate"
+    ),
     // Capcom
     // TODO(cemmer): CPS1, CPS2, CPS3
     // Casio
-    new GameConsole(/PV-?1000/i, [/* '.bin' */], undefined, 'Casio_PV-1000', undefined),
+    new GameConsole(
+      /PV-?1000/i,
+      [
+        /* '.bin' */
+      ],
+      undefined,
+      "Casio_PV-1000",
+      undefined,
+      "pv1000"
+    ),
     // Commodore
-    new GameConsole(/Amiga/i, [], 'amiga', 'Amiga', 'AMIGA'),
-    new GameConsole(/Commodore 64/i, ['.crt', '.d64', '.t64'], undefined, 'C64', 'COMMODORE'),
+    // TODO(dcramer): multiple outputs for Batocera
+    new GameConsole(/Amiga/i, [], "amiga", "Amiga", "AMIGA", undefined),
+    new GameConsole(
+      /Commodore 64/i,
+      [".crt", ".d64", ".t64"],
+      undefined,
+      "C64",
+      "COMMODORE",
+      "c64"
+    ),
     // Coleco
-    new GameConsole(/ColecoVision/i, ['.col'], 'coleco', 'Coleco', 'COLECO'),
+    new GameConsole(
+      /ColecoVision/i,
+      [".col"],
+      "coleco",
+      "Coleco",
+      "COLECO",
+      "colecovision"
+    ),
     // Emerson
-    new GameConsole(/Arcadia/i, [/* '.bin' */], 'arcadia', 'Arcadia', undefined),
+    new GameConsole(
+      /Arcadia/i,
+      [
+        /* '.bin' */
+      ],
+      "arcadia",
+      "Arcadia",
+      undefined,
+      "arcadia"
+    ),
     // Entex
-    new GameConsole(/Adventure Vision/i, [/* '.bin' */], 'avision', 'AVision', undefined),
+    new GameConsole(
+      /Adventure Vision/i,
+      [
+        /* '.bin' */
+      ],
+      "avision",
+      "AVision",
+      undefined,
+      "advision"
+    ),
     // Fairchild
-    new GameConsole(/Channel F/i, [/* '.bin' */], 'channel_f', 'ChannelF', 'FAIRCHILD'),
+    new GameConsole(
+      /Channel F/i,
+      [
+        /* '.bin' */
+      ],
+      "channel_f",
+      "ChannelF",
+      "FAIRCHILD",
+      "channelf"
+    ),
     // GCE
-    new GameConsole(/Vectrex/i, ['.vec'], undefined, 'Vectrex', 'VECTREX'),
+    new GameConsole(
+      /Vectrex/i,
+      [".vec"],
+      undefined,
+      "Vectrex",
+      "VECTREX",
+      "vectrex"
+    ),
     // Interton
-    new GameConsole(/VC ?4000/i, [/* '.bin' */], undefined, 'VC4000', undefined),
+    new GameConsole(
+      /VC ?4000/i,
+      [
+        /* '.bin' */
+      ],
+      undefined,
+      "VC4000",
+      undefined,
+      "vc4000"
+    ),
     // Magnavox
-    new GameConsole(/Odyssey 2/i, [/* '.bin' */], 'odyssey2', 'Odyssey2', 'ODYSSEY'),
+    new GameConsole(
+      /Odyssey 2/i,
+      [
+        /* '.bin' */
+      ],
+      "odyssey2",
+      "Odyssey2",
+      "ODYSSEY",
+      "o2em"
```

Comment:
I'm not entirely convinced this is correct, but the _info.txt files are blank on m ynew batocera install, and they dont really document this. If you trusted docs it'd tell you its "odyssey2", but that doesnt exist on my system and I couldn't find anything else that o2em might be referencing.